deploy/src/main/resources-filtered/production-kubernetes-testing/jobservice-testing-deployment.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: worker-globfilter
+spec:
+  replicas: 1
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        io.kompose.service: worker-globfilter
+    spec:
+      containers:
+      - name: worker-globfilter
+        image: ${worker-globfilter.version}
+        env:
+        - name: CAF_BATCH_WORKER_ERROR_QUEUE
+          value: "${CAF_BATCH_WORKER_ERROR_QUEUE}"
+        - name: CAF_GLOB_WORKER_BINARY_DATA_INPUT_FOLDER
+          value: "${CAF_GLOB_WORKER_BINARY_DATA_INPUT_FOLDER}"
+        - name: CAF_RABBITMQ_HOST
+          value: "${CAF_RABBITMQ_HOST}"
+        - name: CAF_RABBITMQ_PORT
+          value: "${CAF_RABBITMQ_PORT}"
+        - name: CAF_RABBITMQ_PASSWORD
+          value: "${CAF_RABBITMQ_PASSWORD}"
+        - name: CAF_RABBITMQ_USERNAME
+          value: "${CAF_RABBITMQ_USERNAME}"
+        - name: CAF_WORKER_INPUT_QUEUE
+          value: "${CAF_WORKER_GLOBFILTER_INPUT_QUEUE}"
+        resources: {}
+        volumeMounts:
+          - mountPath: /mnt/caf-datastore-root
+            name: worker-datastore
+          - mountPath: /mnt/caf-worker-input-dir
+            name: job-service-demo-input-dir
+      volumes:
+      - name: worker-datastore
+        persistentVolumeClaim:
+          claimName: worker-datastore
+      - name: job-service-demo-input-dir
+        hostPath:
```

Comment:
@rado-straka Not sure, the persistent volume was created by the kompose convert but the other was not added by kompose. I was trying to recreate similar to what I saw in the marathon template so not sure if this is right or not. What is the best solution here?

Reply:
Instead of using pv and pvc, this can be used:
```
           volumeMounts:
        - mountPath: /srv/common/webdav
          name: webdav
      volumes:
      - name: webdav
        hostPath:
          path: /tmp/hostpath_pv/webdav
          type: DirectoryOrCreate
```
This use of volumes is usually used for development not production as it is briefer.
Also, in this case when there is one node(minikube) and one cluster, there is no advantage using pv and pvc.
There is nothing wrong with your solution, my only reason for the comment was to show that it would make less of config files, but no need to change anything.